### ISSUE TITLE
Eliminate need for auxliary getch program

### ISSUE THREAD

**Issue by chipschap** [RogueAI42/shelltris#1]
These small changes will eliminate the need for the external getch auxiliary:

TTYSETTING='raw -echo -icanon time 0 min 0'
function getch()
{  CHAR=$( dd bs=1 count=1 2> /dev/null )  }

And then change all getch calls to simply
getch;

**Comment by DeclanHoare** [comment]
This makes the play area stay completely empty on my machine

**Comment by chipschap** [comment]
Really? I made this change a couple of years ago and it's worked fine, at
least here. I don't know how it could blank the display in the play area,
unless it somehow kills the graphics chars, which doesn't make sense.

It did seem to change the timing calibration, though. I noticed a piece did not appear at the top of the screen for a number of seconds, so if you don't wait a little while it may appear to have an empty play area. I manually edited ~/.config/shelltris/configure.sh to lower the  value of ONE_SECOND by about 40%.

Hope this is the answer ... I kind of like having a single portable file for the game.
